### PR TITLE
Feat/further form UI

### PR DESCRIPTION
**Pull request by pbullhove**
## What does this pull request change?

Since all form objects should have the same look, we refactor the ui into reusable components. 
These are now located under `/shared/`. 

Starting to apply this to the model contained template. 

image

### CODE REVIEW

File: packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx [hunk 1]
```diff
@@ -24,24 +24,43 @@ export const ObjectModelContainedTemplate = (
     uiAttribute?.showExpanded ?? config.showExpanded
   )
   const value = watch(namePath)
-  const isCreated = value && Object.keys(value).length > 0
+  const objectIsNotEmpty = value && Object.keys(value).length > 0
   const canExpand =
-    isCreated &&
+    objectIsNotEmpty &&
```
Comment (soofstad) [inline packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx:29]: Should this also be true if ever "canOpenInTab" is false. I think so

File: packages/dm-core-plugins/src/form/templates/shared/FormExpandedVewWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, { PropsWithChildren } from 'react'
+
+const FormExpandedVewWrapper = ({ children }: PropsWithChildren) => {
```
Comment (soofstad) [inline packages/dm-core-plugins/src/form/templates/shared/FormExpandedVewWrapper.tsx:3]: Typo? uwu

**Review by soofstad — APPROVED** [review]
Good refactor on consolidating styling!

File: packages/dm-core-plugins/src/form/templates/shared/FormExpandedVewWrapper.tsx (hunk 2 above)
Comment (pbullhove) [reply packages/dm-core-plugins/src/form/templates/shared/FormExpandedVewWrapper.tsx:3]: nice catch

File: packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx (hunk 1 above)
Comment (pbullhove) [reply packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx:29]: EO had a pull request fixing this

**Pull request merged**